process/transaction/shardProcess.go

```diff
@@ -664,34 +807,85 @@ func makeUserTxFromRelayedTxV2Args(args [][]byte) *transaction.Transaction {
 }
 
 func (txProc *txProcessor) finishExecutionOfRelayedTxV3(
-	relayerAcnt, acntDst state.UserAccountHandler,
```

Comment:
I don't think we need any "finishExecutionOfRelayedTxV3" this was mainly used for the different execution flow of relayed transactions that were composed by 2 transactions relay tx and user tx.

I would suggest to just go on the flow of regular transactions and change the fee consumption in case the transaction is relayed v3 (create a function for that)

Reply:
updated